types/socketcluster-server/socketcluster-server-tests.ts

```diff
@@ -1,7 +1,10 @@
 // Adapted from README
 
 import http = require('http');
+import agAuth = require('ag-auth');
```

Comment:
this should most probably read:
```ts
import AuthEngine = require('ag-auth');
``` 
https://github.com/SocketCluster/socketcluster-server/blob/ea2175a4d8548ca14bd9b5e7ad2cd3e94d70bc40/server.js#L2